### ISSUE TITLE
screen recording type

### ISSUE THREAD

**Issue by Theselman** [Bnyro/RecordYou#149]
### Describe your suggested feature

"Is it possible to make an example that sends us to the phone start screen and starts the recording from there instead of starting from the counter and application screen when we start the screen recording?"

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Bnyro** [comment]
In theory we could set a countdown of 3 seconds before starting the recording, so that you have a short amount of time to navigate to the home screen. Would that fit your needs?

**Comment by Theselman** [comment]
As soon as the user presses the start recording button, the application will automatically close and the home screen recording will start. In order to finish the recording, he will be able to finish the recording with the help of a panel without entering the application again. actually, I don't want the application screen video to appear both when starting and finishing the recording, I forgot the ending part in the first question. The user may want a record like this example, if there is a way to do this, could you share it? thank you for your answer

**Comment by Theselman** [comment]
While other screen recorders are recording the main menu, there is a panel above and there are options to stop or end the recording without entering the application. I'm wondering how to make these features

**Comment by Bnyro** [comment]
For starting, it works like I described above. For stopping the recording, that's only possible by manually cutting the video and removing the concerning last few seconds.
Though functionality for cutting videos is not planned to be added to Record You.
